### PR TITLE
Add legend for visualize_text

### PR DESCRIPTION
**Pull request by heytitle**
Related to #401.

The PR adds an option for the user to show the legend for `visualize_text`.

image

### CODE REVIEW

File: captum/attr/_utils/visualization.py [hunk 1]
```diff
@@ -543,6 +546,24 @@ def visualize_text(datarecords: Iterable[VisualizationDataRecord]) -> None:
...
         )
 
+    if legend:
+      dom.append('<div style="border-top: 1px solid; margin-top: 5px; \
+        padding-top: 5px; display: inline-block">'
+      )
+      dom.append('<b>Legend: </b>')
+
+      for value, label in zip(
+          [-1, 0, 1], ["Negative", "Neutral", "Positive Attribution"]
```
Comment (NarineK) [inline captum/attr/_utils/visualization.py:556]: Thank you for working on this PR @heytitle ! It feels a little bit that Attribution is related to positive only. Perhaps leaving it just `["Negative", "Neutral", "Positive"]` would be better and the fact that is attribution is intuitive ?

**Comment by heytitle** [comment]
@NarineK thanks for the comment! I haven't thought about that. I thought attribution can be associated to negative and positive.

Anyway, I've updated the PR.

**Comment by heytitle** [comment]
@NarineK anything else we should do here?

**Comment by NarineK** [comment]
Looks great! I'd update the branch. It looks like the branch is out of date :) Thank you :)

**Comment by heytitle** [comment]
There is one failed test due to time exceeded. I'm not sure how I should fix it. Any suggestion?

**Comment by NarineK** [comment]
@heytitle , thank you for the PR! Do you mind updating the branch to fix failing CircleCI!

**Comment by heytitle** [comment]
@NarineK the test didn't succeed because it was timeout. Any suggestion on fixing the issue?

**Comment by NarineK** [comment]
@heytitle , I rerun the tests and it looks like there are some unrelated test errors. We actually fixed them and if you update your branch they should be gone. Do you mind updating the branch with the latest changes ?

**Comment by heytitle** [comment]
@NarineK I guess it's good to be merged now :)

**Pull request closed without merging**